interface/patient_file/front_payment.php

```diff
@@ -946,17 +947,33 @@ function make_insurance() {
 }
 </style>
 <title><?php echo xlt('Record Payment'); ?></title>
+<?php $NameNew = $patdata['fname'] . " " . $patdata['lname'] . " " . $patdata['mname']; ?>
```

Comment:
Is the above $NameNew variable being used?

Reply:
disregard above comment, see it being used below

Reply:
@bradymiller I am not sure what it is doing here as this is redefined on line 166.  It was there in the original script in the <h4></h4> tag and therefore  I left it there :)